src/components/BMDashboard/Issue/Issue.jsx

```diff
@@ -0,0 +1,347 @@
+      currentCheckboxOption = metQualityCheckboxOptions;
+    } else if (selectedOption === 'Labor') {
+      currentCheckboxOption = laborCheckboxOptions;
+    } else if (selectedOption === 'Weather') {
+      currentCheckboxOption = weatherCheckboxOptions;
+    }
+
+    createAndSetCheckboxOptionPairArray(currentCheckboxOption);
+  };
+
+  const handleDropdownChange = option => {
+    setFormData({ ...formData, dropdown: option, checkboxes: new Set(), other: '' });
+    changeCheckboxOption(option);
+  };
+
+  useEffect(() => {
+    // Run this once during initialization to set default dropdown/checkbox options.
+    createAndSetCheckboxOptionPairArray(safetyCheckboxOptions);
+    handleDropdownChange(defaultOption);
+  }, []);
+
+  return (
+    <div className="issue-form-container">
+      <h4 className="issue-title-text">{ISSUE_FORM_HEADER}</h4>
+      <Form>
+        <FormGroup>
+          <Row className="issue-date">
+            <Col className="d-flex justify-content-center">
+              <Label className="sub-title-text">{ISSUE_DATE}</Label>
+            </Col>
+            <Col>
+              <Input
+                className="issue-form-override-css issue-date-input"
+                type="date"
+                name="issueDate"
+                id="issueDate"
+                onChange={e => setFormData({ ...formData, issueDate: e.target.value })}
+              />
+            </Col>
+          </Row>
+        </FormGroup>
+        <Row>
+          <Col>
+            <Label className="red-text">{NOTE}</Label>
+          </Col>
+        </Row>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{ISSUE_TYPE}</Label>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Input
+                className="issue-form-override-css"
+                type="select"
+                name="dropdown"
+                value={formData.dropdown}
+                onChange={e => handleDropdownChange(e.target.value)}
+              >
+                {dropdownOptions.map(option => (
+                  <option key={option} value={option}>
+                    {option}
+                  </option>
+                ))}
+              </Input>
+            </Col>
+          </Row>
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{CONSEQUENCES_TITLE}</Label>
+            </Col>
+          </Row>
+          {checkboxOptions.map(pair => (
+            <Row key={pair} className="consequences-checkboxes">
+              {pair.map(option => (
+                <Col key={option}>
+                  <Label check>
+                    <Input
+                      type="checkbox"
+                      checked={formData.checkboxes.has(option) || false}
+                      onChange={() => handleCheckboxChange(option)}
+                    />
+                    {option}
```

Comment:
Could we please change the cursor to a pointer when hovering over the option text as well as the checkbox? This will improve the user experience by making it clear that the option text is also clickable.